liferay-echo/liferay-portal
# LPS-192600 Show folders in Display Page Templates tab

Motivation
=======
The new feature to display and edit object entries is producing that users have several display pages for the same object. This said, it will be very useful to provide a way to our users so they can organise them.

Proposed Solution
========
Provide the ability to create folders in “Display Page Templates” section.

Steps to Verify
========
1. Go to Design > Page Templates > Display Page Templates tab
2. Create several folders and DPTs
3. Ensure that are show un separate sections like in the Doc Library (see image attached)

Screenshots 
========
<img width="1385" alt="DPT_Folders" src="https://github.com/liferay-echo/liferay-portal/assets/85171101/0495a08f-fa8b-4204-9c72-5b9813c62e4a">


COMMITS:
- LPS-192600 Add method to retrieve DisplayPageCollections and LayoutPageTemplateEntries in the same search
- LPS-192600 Add method to retrieve DisplayPageCollections and LayoutPageTemplateEntries count in the same search
- LPS-192600 buildService
- LPS-192600 Implementation of methods to retrieve and count DisplayPageCollections and LayoutPageTemplateEntries in the same search
- LPS-192600 Comparators between LayoutPageTemplateCollections and LayoutPageTemplateEntries
- LPS-192600 Generifying searchContainer to retrieve collections and entities
- LPS-192600 Create horizontal card for DisplayPageTemplateCollections
- LPS-192600 Update show items in Display Page Templates tab
- LPS-192600 Create Layout Page template row splitter to show folders and DPTs grouped in different sections
- LPS-192600 Date formatter
